package.json

```diff
@@ -49,57 +49,53 @@
     ]
   },
   "devDependencies": {
-    "@babel/core": "^7.13.16",
-    "@storybook/addon-actions": "^6.3.0-alpha.14",
-    "@storybook/addon-essentials": "^6.3.0-alpha.14",
-    "@storybook/addon-links": "^6.3.0-alpha.14",
-    "@storybook/builder-webpack5": "^6.3.0-alpha.14",
-    "@storybook/react": "^6.3.0-alpha.14",
+    "@babel/core": "^7.16.7",
+    "@storybook/builder-webpack5": "^6.3.12",
+    "@storybook/manager-webpack5": "^6.3.12",
+    "@storybook/react": "^6.3.12",
```

Comment:
I chose not to upgrade to Storybook v6.4, as I’ll be spending much more time with this in a subsequent PR.